charts/gardener-extension-provider-kubevirt/templates/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "name" . }}
+  namespace: {{ .Release.Namespace }}
+  {{-  if .Values.ignoreResources }}
+  annotations:
+    resources.gardener.cloud/ignore: "true"
+  {{- end }}
```

Comment:
done

Reply:
btw. this feature didn't work for me, I saw that controller was still reconciling some CRs, is this annotation being read by extension pkgs? Like the main reconciliation loops for infra, controlplane and worker CRs?

Reply:
What exactly are you trying to achieve? The annotation is meant for the so called "gardener resource manager" when it tries to reconcile "managed resources". The extension controller itself is managed in the seed as such a resource, so this annotation would prevent GRM from reconciling it. Is this really what you want, and why? I didn't comment much on this so far because I thought it could be addressed later, and I haven't had the chance to test the extension yet.

Reply:
I wanted to use it for the local setup, it means I would have to deploy the controller registration CR (which is required) with the annotation and later start the extension controller locally from my desktop (`make start`). I found some documentation that says with the annotation the controller would not pick infra, control plane, worker CRs.

I decided to remove that annotation for now as I need to understand better the entire flow you described, and when I have a ready solution I will open a new PR for local dev setup. Maybe the doc I found will required an update too (I have to find that doc as I don't remember its location), if yes then I will create a PR there too.

Reply:
I think the purpose of the annotation is not what you had understood, see above. If you want to avoid reconciles from the cluster pods, the easiest way to achieve is to scale the extension deployment to 0 by setting the `replicaCount` value in the `controllerregistration` resource to 0.